core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,28 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          var POSSIBLE_TITLES = ['Hulk Neuroscience',
+                                 'Quantum Starks',
+                                 'Wonder Anatomy',
+                                 'Elvish, language of "Lord of the Rings',
+                                 'The Science of Superheroes'
+          ];
+          $scope.setStatusMessage('Processing...');
+          for (var i = 0; i < $scope.DummyExplorationCount; ++i) {
+            var randomTitle = Math.floor(
+                              Math.random() * POSSIBLE_TITLES.length
```

Comment:
This should be indented by 2 spaces from the previous line. You can move the ");" from the next line to the end of this one.